src/main/resources/reproduce/msmarco-v2.1-doc.yaml

```diff
@@ -0,0 +1,39 @@
+conditions:
```

Comment:
Combine doc with doc-segmented?

Reply:
While coding I was thinking how RunBeir could just simply be another command line option in RunMsMarco, due to the uniform interface in the yaml files. Would it make sense if we combine them into one script, called "RunRepro" or something similar, and specify the different regressions in yaml files?

Reply:
Currently, in Pyserini, each corpus is a separate file: https://github.com/castorini/pyserini/tree/master/pyserini/2cr

We're modelling after that repo as a start, but can probably further combine later?

For now, let's have MS MARCO and BEIR as separate classes, and then maybe circle back?